scripts/generate_scenario.py

```diff
@@ -27,18 +27,27 @@ def generate_scenario(runtime_fn, features_fn, cutoff):
                    "features_cutoff_memory": "?",
                    "algorithms_deterministic": "?", #TODO
                    "algorithms_stochastic": "",
+                   "metainfo_algorithms": {},
                    "features_deterministic": "?", #TODO
                    "features_stochastic": "",
                    "number_of_feature_steps": 1,
                    "feature_steps":{"ALL":{"provides":"?"}}, #TODO
                    "default_steps": ["ALL"]}
+    
+    #default algorithm info
+    algo_info = {"configuration": "",
+                 "deterministic": "true"}
```

Comment:
If I'm not wrong, this should be a Boolean (True)

Reply:
When I put "deterministic": [True], it prints 
deterministic:
- true.
But in all the scenarios I saw, the field looks like:
deterministic: true
which I could only achieve by putting true as a string. So, I made it for cosmetic reasons only:)